tests/FakeItEasy.Specs/SelfInitializedFakesSpecs.cs

```diff
@@ -142,12 +436,13 @@ public static class SelfInitializedFakesSpecs
 
                         countDuringPlayback = playbackFakeService.GetCount("8");
                     }
-                });
+                })
+                .Teardown(() => File.Delete(fileRecorderPath));
 
-            "Then the recording fake returns the wrapped service's result"
+            "Then it should return the expected result while recording"

```

Comment:
Do we want to remove "should" before merging? Given our policy against using it, this change (and the one below) is a slight regression.